.github/workflows/helm-test.yaml

```diff
@@ -0,0 +1,77 @@
+#################################################################################
+# Copyright (c) 2022,2023 T-Systems International GmbH
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+
+name: "Lint and Test Charts"
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - 'charts/**'
+    branches:
+      - main
+      - 'qg4-check'
+      
+jobs:
+    lint-test:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+              with:
+                fetch-depth: 0
+
+            - name: Kubernetes KinD Cluster
+              uses: container-tools/kind-action@v1
+        
+            - name: Build image
+              uses: docker/build-push-action@v3
+              with:
+                context: .
+                push: true
+                tags: ghcr.io/catenax-ng/tx-sd-factory/sdfactory:test 
```

Comment:
You shouldn't tag and push to the ghcr package manager, but instead to the local docker registry in the kind cluster like in this: `kind-registry:5000/sd-factory:testing`

Reply:
Done